ParselyTracker/ParselyTracker.swift

```diff
@@ -212,7 +212,11 @@ public class Parsely {
     /// After given `seconds`, invoke the given target-action in the event processor queue.
     func scheduleEventProcessing(inSeconds seconds: Double, target: AnyObject, selector: Selector) -> Cancellable {
         Just(0)
-            .delay(for: .seconds(seconds), scheduler: eventProcessor)
+            // From the Apple documentation it's not clear what the default tolerance is.
+            // They have an example with a .seconds(3) delay saying "3 seconds (±0.5 seconds)".
+            // To say on the safe side, we specify a 1ms tolerance, meaning "the Delay publisher may
+            // deliver elements this much sooner or later than the interval specifies."
+            .delay(for: .seconds(seconds), tolerance: .milliseconds(1), scheduler: eventProcessor)
```

Comment:
See discussion at https://github.com/Automattic/AnalyticsSDK-iOS/pull/6#pullrequestreview-1387210695 regarding how this parameter affects (or fails to) the runtime behavior we experience in the tests.

I guess if the experience is that it doesn't help making the tests (and therefore the real world behavior) more predictable one could argue there's no point in adding this. 🤔 I decided to add it as a "just in case" measure. Keen to hear what you think.